docs/source/user/gts-headers-in-wis2.rst

```diff
@@ -0,0 +1,63 @@
+.. _gts-headers-in-wis2:
+
+Optional: adding GTS headers to WIS2 notifications during the transition period
+===============================================================================
+
+Overview
+--------
+
+This section provides guidance how to add GTS headers to your WIS2 notifications.
+
+By adding GTS headers to your WIS2 notifications, you can stop your MSS and still have your data available on the GTS during the transition period.
+
+To enable the WIS2 to GTS Gateway to correctly identify the data to be republished on the GTS, you need to include the GTS property in the WIS2 Notification Message as follows:
+
+```json
+{
+  "properties": {
+    "gts": {
+      "ttaaii": "FTAE31",
+      "cccc": "VTBB"
+    }
+  }
+}
+```
+
+The wis2box can add these to your WIS2 Notifications automatically, provided you specify the additional file `gts_headers_mappings.json` that contains the required information to map the GTS headers to the incoming filenames.
+
+Note that this is optional and only required if you want to turn off the system responsible for sending data to the GTS during the transition period.
+
+gts_headers_mappings.json
+-------------------------
+
+If you want to add GTS headers to your WIS2 notifications, you need to create a JSON file that maps the GTS headers to the incoming filenames. 
```

Comment:
Is this configuration JSON or CSV?  The text here shows JSON but `data/gts_headers_mapping.csv` says otherwise.